cli/cmd/capture/create.go

```diff
@@ -443,32 +436,32 @@ func deleteJobs(kubeClient kubernetes.Interface, jobs []batchv1.Job) []string {
-	createCapture.Flags().StringVar(&tcpdumpFilter, "tcpdump-filter", "", "Raw tcpdump flags which works only for Linux")
-	createCapture.Flags().StringVar(&excludeFilter, "exclude-filter", "", "A comma-separated list of IP:Port pairs that are "+
+	createCapture.Flags().StringVar(&s3Bucket, "s3-bucket", captureConstants.DefaultS3Bucket, "Bucket in which to store capture files")
+	createCapture.Flags().StringVar(&s3Path, "s3-path", captureConstants.DefaultS3Path, "Prefix path within the S3 bucket where captures will be stored")
+	createCapture.Flags().StringVar(&s3AccessKeyID, "s3-access-key-id", captureConstants.DefaultS3AccessKeyID, "S3 access key id to upload capture files")
+	createCapture.Flags().StringVar(&s3SecretAccessKey, "s3-secret-access-key", captureConstants.DefaultS3SecretAccessKey, "S3 access secret key to upload capture files")
+	createCapture.Flags().StringVar(&tcpdumpFilter, "tcpdump-filter", captureConstants.DefaultTcpdumpFilter, "Raw tcpdump flags which works only for Linux")
+	createCapture.Flags().StringVar(&excludeFilter, "exclude-filter", captureConstants.DefaultExcludeFilter, "A comma-separated list of IP:Port pairs that are "+
 		"excluded from capturing network packets. Supported formats are IP:Port, IP, Port, *:Port, IP:*")
-	createCapture.Flags().StringVar(&includeFilter, "include-filter", "", "A comma-separated list of IP:Port pairs that are "+
+	createCapture.Flags().StringVar(&includeFilter, "include-filter", captureConstants.DefaultIncludeFilter, "A comma-separated list of IP:Port pairs that are "+
 		"used to filter capture network packets. Supported formats are IP:Port, IP, Port, *:Port, IP:*")
-	createCapture.Flags().BoolVar(&includeMetadata, "include-metadata", true, "If true, collect static network metadata into capture file")
-	createCapture.Flags().IntVar(&jobNumLimit, "job-num-limit", 0, "The maximum number of jobs can be created for each capture. 0 means no limit")
-	createCapture.Flags().BoolVar(&nowait, "no-wait", true, "Do not wait for the long-running capture job to finish")
-	createCapture.Flags().BoolVar(&debug, "debug", false, "When debug is true, a customized retina-agent image, determined by the environment variable RETINA_AGENT_IMAGE, is set")
+	createCapture.Flags().BoolVar(&includeMetadata, "include-metadata", captureConstants.DefaultIncludeMetadata, "If true, collect static network metadata into capture file")
+	createCapture.Flags().IntVar(&jobNumLimit, "job-num-limit", captureConstants.DefaultJobNumLimit, "The maximum number of jobs can be created for each capture. 0 means no limit")
+	createCapture.Flags().BoolVar(&nowait, "no-wait", captureConstants.DefaultNowait, "Do not wait for the long-running capture job to finish")
+	createCapture.Flags().BoolVar(&debug, "debug", captureConstants.DefaultDebug, "When debug is true, a customized retina-agent image, determined by the environment variable RETINA_AGENT_IMAGE, is set")
```

Comment:
It's great to define constants for the default values, however for the ones that don't have any defaults, let's either leave it as is or just create single const for default string, bool and int and use them.

Reply:
Got rid of all the empty defaults